src/common/objects/CircleRigidBody.cpp

```diff
@@ -72,4 +72,18 @@ sf::Vector2f CircleRigidBody::getCenter() const {
     sf::Vector2f position = getPosition();
     float radius = getRadius();
     return sf::Vector2f(position.x + radius, position.y + radius);
-}
\ No newline at end of file
+}
+
+void CircleRigidBody::hermiteInterpolation(sf::Vector2f position, sf::Vector2f velocity, float a,
```

Comment:
it would be cool if you could add a summary here explaining what it's doing as a comment

Reply:
it's hard to explain and I dont really understand it well
i expect the reader to just google "hermite interpolation" to figure it out